<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskFlow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="category_filter_combobox"/>
    </item>
    <item>
     <widget class="QListWidget" name="task_list_widget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add_task_button">
        <property name="text">
         <string>➕ Nueva</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="edit_task_button">
        <property name="text">
         <string>✏️ Editar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_task_button">
        <property name="text">
         <string>❌ Eliminar</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="toggle_timer_button">
        <property name="text">
         <string>▶️ Iniciar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="action_estadisticas"/>
    <addaction name="action_exportar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <action name="action_estadisticas">
   <property name="text">
    <string>Estadísticas</string>
   </property>
  </action>
  <action name="action_exportar">
   <property name="text">
    <string>Exportar CSV</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
